testcases/kernel/syscalls/mmap/mmap09.c

```diff
@@ -1,119 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 /*
  * Copyright (c) International Business Machines  Corp., 2003
- *
- * This program is free software;  you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY;  without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
- * the GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program;  if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ * HISTORY
+ *	04/2003 Written by Paul Larson
+ * Copyright (C) 2024 SUSE LLC Andrea Manzini <andrea.manzini@suse.com>
  */
 
-/*
- * Test Description:
+/*\
+ * [Description]
  *  Verify that truncating a mmaped file works correctly.
  *
  * Expected Result:
  *  ftruncate should be allowed to increase, decrease, or zero the
  *  size of a file that has been mmaped
  *
- *  Test:
- *   Use ftruncate to shrink the file while it is mapped
- *   Use ftruncate to grow the file while it is mapped
- *   Use ftruncate to zero the size of the file while it is mapped
+ *  Test - Use ftruncate to:
+ *   1. shrink the file while it is mapped
+ *   2. grow the file while it is mapped
+ *   3. zero the size of the file while it is mapped
```

Comment:
This will be badly formatted (more spaces after `*` will behave like html `<code>`. Something like this would be better:

```C
/*\
 * [Description]
 *
 * Verify that truncating a mmaped file works correctly.
 *
 * Use ftruncate to:
 *
 * 1. shrink the file while it is mapped
 * 2. grow the file while it is mapped
 * 3. zero the size of the file while it is mapped
 */